src/classes/groundplaces.ts

```diff
@@ -272,15 +324,27 @@ export class GroundPlacesController {
 
       throw new Error(error.message);
     }
+
+    this.storageService.addGroundPlaceDiff(stopClusterGpuid, {
+      type: ActionType.UPDATE_STOP_CLUSTER,
+      params: {
+        ...propertiesToUpdate,
+      },
+    });
   }
 
   /**
    * @description Add a StopGroup to a StopCluster.
    * @param {StopGroupGpuid} stopGroupGpuidToAdd - Ground place unique identifier of the StopGroup to add.
    * @param {StopClusterGpuid} intoStopClusterGpuid - Ground Place unique identifier of the new StopCluster parent.
+   * @param {CreateDiff} shouldCreateDiff - True by default, if you don't want to have diff from this method, you can deactivate it.
```

Comment:
No default values to prefer explicit usage ? @Theo-Le-Donne @rimiti @Gaspardloriot

Reply:
also naming is a bit fuzzy.

"logToActionHistory" may be ?

Reply:
hmm, yes you're right i think, it's probably better to remove the default value to be sure of what it's doing.
Naming is to be refact accordnig to https://github.com/tictactrip/ground-places-sdk/pull/9#discussion_r557409694